hotham-simulator/src/inputs.rs

```diff
@@ -0,0 +1,37 @@
+use std::collections::HashSet;
+
+use winit::event::{ElementState, VirtualKeyCode};
+
+#[derive(Debug, Clone)]
+pub enum HothamInputEvent {
```

Comment:
nitpick: This type doesn't add much beyond winit's - do we really need a custom implementation?

Reply:
Yeah good point, much better now its removed